telegram-bot/src/main/java/pro/sky/telegrambot/listener/TelegramBotUpdatesListener.java

```diff
@@ -18,6 +21,8 @@ public class TelegramBotUpdatesListener implements UpdatesListener {
 
     @Autowired
     private TelegramBot telegramBot;
+    @Autowired
+    private WritingToTheDatabaseImpl writing;
```

Comment:
Зависимости строим на абстракциях (интерфейсах)
```suggestion
    private WritingToTheDatabase writing;
```